test/conformance/ingress/README.md

```diff
@@ -0,0 +1,119 @@
+# Ingress Conformance Testing
+
+This directory contains Ingress conformance tests for Knative Ingress resource. 
+
+## Environment requirements
+
+### Development tools
+
+1. [`go`](https://golang.org/doc/install): The language `Knative Serving` is
+   built in (1.13 or later)
+1. [`ko`](https://github.com/google/ko): Build tool to setup the environment.
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
+   managing development environments.
+
+### Test environment
+
+1. [A running `Knative Serving` cluster.](https://github.com/knative/serving/blob/master/DEVELOPMENT.md#prerequisites),
+   with the Ingress implementation of choice installed.
+   ```bash
+   # Set the Ingress class annotation to use in tests.
+   # Some examples:
+   #   export INGRESS_CLASS=gloo.ingress.networking.knative.dev      # Gloo Ingress
+   #   export INGRESS_CLASS=istio.ingress.networking.knative.dev     # Istio Ingress
+   #   export INGRESS_CLASS=kourier.ingress.networking.knative.dev   # Kourier Ingress
+   export INGRESS_CLASS=<your-ingress-class-annotation>
+   ```
+1. Knative Serving source code check out at `${SERVING_ROOT}`. Often this is
```

Comment:
The test images are also moved here. We probably need a new `upload-test-images.sh`

Reply:
Ah! Sounds good! I didn't realize that, let me make the changes and come back.

Reply:
Updated!